### ISSUE TITLE
Regression with the token PRECEDING with version 10 for bigquery formatting

### ISSUE THREAD

**Issue by phbou72** [sql-formatter-org/sql-formatter#415]
**Describe the bug**
There seems to be a regression in what is supported by sql-formatter for the bigquery SQL language.

**Expected behavior**
I expected the version 10 to still support the token PRECEDING.

**Actual behavior**

I got the following error : 
<img width="1448" alt="187225626-99e10276-3d9e-49a1-914e-4015bfc19f6d" src="https://user-images.githubusercontent.com/1980396/187443697-c61a7e49-a384-467d-9992-06cd61eec8e3.png">

**Usage**

- How are you calling / using the library?
```
format(sql, {
    language: 'bigquery',
});
```
- What SQL language(s) does this apply to?
 bigquery
- Which SQL Formatter version are you using?
10

**Comment by nene** [comment]
Thanks for reporting.

Could you please also include the exact SQL that triggers this error?

**Comment by phbou72** [comment]
Sure, here it is: 

```
WITH
  `base_0` AS (
    SELECT
      `date_datetime` AS `day`,
      SUM(`amount`) AS `amount`,
      0 AS `sort_order`
    FROM
      `table.table__amount`
    GROUP BY
      `date_datetime`
  ),
  `rollup_0` AS (
    SELECT
      `day`,
      `sort_order`,
      AVG(`amount`) OVER (
        PARTITION BY
          `explosion`
        ORDER BY
          `day` ROWS BETWEEN 6 PRECEDING
          AND CURRENT ROW
      ) AS `amount`
    FROM
      `base_0`
  ),
  `dataset_0` AS (
    SELECT
      *
    FROM
      `rollup_0`
    WHERE
      (
        (`day` >= datetime("2022-01-01"))
        AND (`day` <= datetime("2022-08-26"))
      )
  )
SELECT
  ROUND((`amount`), 2) AS `value`,
  `day`,
  `sort_order`
FROM
  `dataset_0`
ORDER BY
  `sort_order`,
  `day`

```

I had to replace a few thing tho because we can't leak the schema to the public.

**Comment by nene** [comment]
Released the fix in 10.1.1

**Comment by phbou72** [comment]
Thank you!
